tests/test_acm.py

```diff
@@ -59,3 +59,23 @@ def test_certificate_delete_in_use_error(self):
         client = factory().client("acm")
         arn = "arn:aws:acm:us-west-2:644160558196:certificate/b867707e-33c3-4024-b45d-b6133c3b4c05"
         self.assertTrue(client.get_certificate(CertificateArn=arn))
+
+    def test_certificate_tag_untage_mark(self):
+        factory = self.replay_flight_data("test_certificate_tag_untage_mark")
```

Comment:
s/untage/untag ? should be able to just move the directory and rename here.

Reply:
Sorry, typo! Fixed now